src/shell/services/announcements.ts

```diff
@@ -0,0 +1,52 @@
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+
+import { Announcement } from "./types";
+
+export const announcementsApi = createApi({
+  reducerPath: "announcementsApi",
+  baseQuery: fetchBaseQuery({ baseUrl: "https://www.zesty.io/-/instant/" }),
```

Comment:
Updated. Renamed service to marketingApi so that it's more generic towards any marketing-related queries. Also made the query have the model zuid by default instead of passing it as a prop